public/util/data.ts

```diff
@@ -584,7 +585,10 @@ export function getVariableSummariesByState(
       // prioritize FI over MI
       sortedVariables = sortVariablesByImportance(sortedVariables);
     }
-
+    // remove any pattern cluster variables
+    sortedVariables = sortedVariables.filter((sv) => {
```

Comment:
I think that filtering should be done on line 583 when `sortedVariables` is created.
Like this we make sure that the cluster variables are not ranked by mistakes, etc.

Reply:
Good point.